packages/unpublished-eslint-rules/src/rules/prefer-type-only-import.js

```diff
@@ -260,7 +260,7 @@ function lintDeclarationForTypeOnlyImports(declaration, usageData, context) {
 }
 
 module.exports = {
-  name: 'prefer-type-only-import',
+  name: 'prefer-static-type-import',
```

Comment:
Did we want to be making this change?

Reply:
Yep!  b/c now it will lint the fact that you only should use `import type` instead of just `type`.

Reply:
isn't that what the other new file is for?

Reply:
https://github.com/emberjs/data/pull/7608/files#diff-8eb7a08d0916321018b046fbe96f871b36b23290fcf0f521ebea101e8fb58f2e

Reply:
Are you sure we don't want to delete this one?

Reply:
Yes you are right.  Fixed!